src/block-experiments-setup.js

```diff
@@ -3,23 +3,9 @@
  */
 // This file is to set up the jetpack/layout-grid block that currently lives in block-experiments/blocks/layout-grid
 import { registerBlock } from '../block-experiments/blocks/layout-grid/src';
-import setupLocale from './setup-locale';
-import { getTranslation as getLayoutGridTranslation } from './i18n-translations/layout-grid';
-
-const LAYOUT_GRID_LOCALE_DOMAIN = 'layout-grid';
 
 export function setupBlockExperiments( capabilities ) {
 	if ( capabilities.layoutGridBlock ) {
 		registerBlock();
 	}
 }
-
-export function setupBlockExperimentsLocale( locale, extraTranslations ) {
-	// Setup locale for Layout Grid
-	setupLocale( {
-		domain: LAYOUT_GRID_LOCALE_DOMAIN,
-		locale,
-		extraTranslations,
-		getTranslation: getLayoutGridTranslation,
-	} );
-}
```

Comment:
The translations for the layout grid block are now included in the index file when registering Gutenberg ([reference](https://github.com/wordpress-mobile/gutenberg-mobile/blob/504270493676fe9c2a219519bb70a8d76106dee1/src/index.js#L12-L21)).